app/controllers/widgets_controller.rb

```diff
@@ -4,6 +4,7 @@ class WidgetsController < ApplicationController
   # GET /widgets
   # GET /widgets.json
   def index
+    logger.info "I am a really long line that goes and goes and goes and goes!!!!"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.